app/views/pre_registrations/edit.html.erb

```diff
@@ -1,132 +1,156 @@
-            </p>
-            <p>
-                <%= f.check_box :NdaSigned, :checked => @pre_registration.NdaSigned %>
-            </p>
-        </div>
+            <div class="col-md-6">
+                <%= f.label :NdaSigned, 'NDA Signed' do %>
+                    <%= 'NDA Signed' %>
+                    <span class='required-asterisk'>*</span>
+                    <%= f.check_box :NdaSigned, :checked => @pre_registration.NdaSigned, :required => true %>
+                    <% if !@pre_registration.ndaDoc.present? %>    
+                        <%= f.file_field :ndaDoc %>
+                    <% end %>
+                <% end %>
+            </div>
 
-        <div class="col-md-6">
-            <p>
-                <%= label :workstationReady, 'Workstation Ready' %>
-            </p>
-            <p>
-                <%= f.check_box :workstationReady, :checked => @pre_registration.workstationReady %>
-            </p>
-        </div>
+            <div class="col-md-6">
+                <%= f.label :workstationReady, 'Workstation prepared for new employee?' do %>
+                    <%= 'Workstation prepared for new employee? ' %>
+                    <span class='required-asterisk'>*</span>
+                    <%= f.check_box :workstationReady, :checked => @pre_registration.workstationReady, :required => true %>
+                <% end %>
+            </div>
+
+            <div class="col-md-6">
+                <%= f.label :packReady, 'Greeting pack prepared for new employee?' do %>
+                    <%= 'Greeting pack prepared for new employee?' %>
+                    <span class='required-asterisk'>*</span>
+                    <%= f.check_box :packReady, :checked => @pre_registration.packReady, :required => true %>
+                <% end %>
+            </div>
 
-        <div class="col-md-6">
-            <p>
-                <%= label :packReady, 'Pack Ready' %>
-            </p>
-            <p>
-                <%= f.check_box :packReady, :checked => @pre_registration.packReady %>
-            </p>
         </div>
-    </div>
     <br/>
+    <hr>
     <%= hidden_field_tag 'selected', @is_for_admin  %>
     <br/>
+    <div class="row">
+        <%if @is_for_admin =="23" %>
+            <div class="col-md-6">
+                <%= f.label :companyEmail, 'Company Email' do %>
+                    <%= 'Company Email' %>
+                    <span class='required-asterisk'>*</span>
+                    <%= f.text_field :companyEmail, :class => 'form-control', :maxlength=>"128", :required => true %>
+                <% end %>
+            </div>
+            <div class="col-md-6">
+                <%= f.label :HR_email, 'Notify HR to follow the next step' do %>
+                    <%= 'Notify HR to follow the next step' %>
+                    <span class='required-asterisk'>*</span>
+                    <%= f.text_field :HR_email, :class => 'form-control', :maxlength=>"128", :required => true %>
```

Comment:
Change the hash syntax

Reply:
changed apu
please check it once again just to make sure whether I've used proper syntax or not